apps/crn-server/src/validation/research-output.validation.ts

```diff
@@ -18,6 +19,32 @@ type ResearchOutputParameters = {
   researchOutputId: string;
 };
 
+type ResearchOutputDraftParameters = FetchPaginationOptions & {
+  status: string;
```

Comment:
what is entityId and also why does it only apply to drafts? I take it you want to say that only should they be looking for drafts do we want to allow searching by some entityId? 

I would just use the basic research output query string validation and add the status property (and maybe the entityId - whatever that is - if it's a working-group ID then I guess you should name it that)

Reply:
entityId is either working group or team id. It only applies to drafts as we want only members of a team or working group to view the drafts.

Reply:
pretty sure this validation logic is used for just that endpoint and i actually think you should just expand the existing validation to add support for filtering by status and by a workingGroupId and by a teamId. I think we want to be explicit so I would add all three filters.

In fact I wouldnt bother tying it to just drafts, i would just expose such filter for the entire search. I think implementation-wise this will actually make it a lot easier.